Auth0/CredentialsManager.swift

```diff
@@ -76,15 +76,17 @@ public struct CredentialsManager {
     }
     #endif
 
+    #if WEB_AUTH_PLATFORM
     /// Enable Biometric Authentication for additional security during credentials retrieval
     ///
     /// - Parameters:
     ///   - title: main message to display when Touch ID is used
     ///   - cancelTitle: cancel message to display when Touch ID is used (iOS 10+)
     ///   - fallbackTitle: fallback message to display when Touch ID is used after a failed match
-    #if WEB_AUTH_PLATFORM
-    public mutating func enableBiometrics(withTitle title: String, cancelTitle: String? = nil, fallbackTitle: String? = nil) {
-        self.bioAuth = BioAuthentication(authContext: LAContext(), title: title, cancelTitle: cancelTitle, fallbackTitle: fallbackTitle)
+    ///   - evaluationPolicy: policy to be used for authentication policy evaluation
+    @available(iOS 9.0, macOS 10.15, *)
```

Comment:
This would be required if we want to pass `LAPolicy`. But given that most of functionality of `BioAuthentication` struct is limited to these system versions, I think this should be ok.

Alternative would be to just have separate method with additional parameter, let me know if I should add it, if current solution is not sufficient.

Reply:
The current solution is fine.